modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -70,20 +41,20 @@ You can also change the encryption algorithm and crypto mode by setting addition
 | The type of algorithm you use to encrypt the content of the property. See <<supported_algorithms>> for a complete list.
 
 | Mode
-| The procedure that allows the Mule runtime to repeatedly use a block cipher with a single key. See <<supported_modes>> for a full list.
+| The procedure that allows the Mule runtime engine to repeatedly use a block cipher with a single key. See <<supported_modes>> for a full list.
 |===
 
-[[supported_files]]
-== Supported Files
+== Create a Secure Configuration Properties File
 
-The Secure Configuration Properties module supports both YAML configuration files and Spring-formatted properties files, as shown in the examples below. The recommended approach is to use a YAML configuration file, because it allows the addition of type validations and autocompletion.
+The first task in securing configuration properties is to create a `.yaml` configuration file or a Spring-formatted `.properties` file, where you will define the properties. The recommended approach is to use a YAML configuration file, because it allows the addition of type validations and autocompletion. The Mule Secure Configuration Properties Extension module enables you to configure these `.yaml` or `.properties` file types. +
```

Comment:
The recommended approach is to use > MuleSoft recommends using

Reply:
Updated.